src/Illuminate/Notifications/Channels/HttpChannel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Illuminate\Notifications\Channels;
+
+use GuzzleHttp\Client as HttpClient;
+
+abstract class HttpChannel
+{
+    /**
+     * The HTTP client instance.
+     *
+     * @var \GuzzleHttp\Client
+     */
+    protected $http;
+
+    /**
+     * Create a new HTTP channel instance.
+     *
+     * @param  \GuzzleHttp\Client  $http
+     * @return void
+     */
+    public function __construct(HttpClient $http)
+    {
+        $this->http = $http;
+    }
+}
```

Comment:
It makes more sense the extending classes to implement the interface
As this abstract class will only be extended by HTTP related Channels